### ISSUE TITLE
how to setting in burpsuite 2.0

### ISSUE THREAD

**Issue by sfdota** [bugcrowd/HUNT#57]
how to setting in burpsuite 2.0 scaner??

**Comment by adamczi** [comment]
Hi, I'd also (I guess?) like to report, that I cannot make Hunt work in latest BurpSuite. Installation went fine, but there is no "Send to HUNT" or similar option in context menu, nor new requests show up in Hunt tabs. 

Is the project discontinued and won't work on new Burp?
Best regards

**Comment by iNoSec2** [comment]
weird, try to delete your extension and retry because at least you must have hunt methodology working ;)
Even the scanner work for me on 2020.4.1, so the latest and it worked on the previous version too

**Comment by cak** [comment]
Hey @sfdota and @iNoSec2, just wanted to let you know the HUNT Scanner is now in the BApp Store for Burp v2.0 - https://portswigger.net/bappstore/059343223d094d16a0a8440485bc5c5e

**Comment by iNoSec2** [comment]
> Hey @sfdota and @iNoSec2, just wanted to let you know the HUNT Scanner is now in the BApp Store for Burp v2.0 - https://portswigger.net/bappstore/059343223d094d16a0a8440485bc5c5e

Nope that's the remix which doesnt match all the original does, and the original works perfectly with java 11 to 13, just need to load 2-3 Times sometimes

**Comment by cak** [comment]
Yes iNoSec2, you are right it is the remix (HUNT Scanner v2) - is there anything that we can add to make it better or include features that you are missing?
